<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>359</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>缺货商品:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>当前商品：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>621</width>
        <height>131</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>库存浏览</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>621</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查看库存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="printbtn">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打印库存清单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findbtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查找</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="information">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>550</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>商品销售情况</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="list_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>731</width>
        <height>261</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="printbtn_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>380</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打印销售清单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="totalbtn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>450</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>统计</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>455</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>总收入情况：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="totaltext">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>450</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="incar">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>75</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加至购物车</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>已购商品：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="cargoods">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>445</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>共计</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="totalprice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>440</y>
        <width>104</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="paytotal">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>440</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>结算</string>
      </property>
     </widget>
     <widget class="QPushButton" name="printbtn_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>打印购物清单</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="number">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>19</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>消费结算</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>631</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="findbtn_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查找商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebtn">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除商品</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>条形码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goodname_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="barcode_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="outprice">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inprice">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="sup_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>230</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>进价：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>180</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>280</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>售价：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>供货商：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>商品信息修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>92</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>190</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>商品数量:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>240</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>进货价格：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>290</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>销售价格：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>340</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>供货商：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="goodname">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="barcode">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inPrice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="outPrice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="input">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>入库</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>140</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>条形码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sup">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>商品入库</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="incontrol">
    <property name="title">
     <string>进货管理</string>
    </property>
    <addaction name="ingood"/>
   </widget>
   <widget class="QMenu" name="stockcontrol">
    <property name="title">
     <string>库存管理</string>
    </property>
    <addaction name="stock"/>
    <addaction name="change"/>
   </widget>
   <widget class="QMenu" name="incomecontrol">
    <property name="title">
     <string>销售管理</string>
    </property>
    <addaction name="income"/>
   </widget>
   <widget class="QMenu" name="salecontrol">
    <property name="title">
     <string>收银台</string>
    </property>
    <addaction name="sale"/>
   </widget>
   <addaction name="incontrol"/>
   <addaction name="stockcontrol"/>
   <addaction name="incomecontrol"/>
   <addaction name="salecontrol"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ingood">
   <property name="text">
    <string>进货</string>
   </property>
  </action>
  <action name="stock">
   <property name="text">
    <string>库存查看</string>
   </property>
  </action>
  <action name="income">
   <property name="text">
    <string>账目</string>
   </property>
  </action>
  <action name="sale">
   <property name="text">
    <string>结算</string>
   </property>
  </action>
  <action name="change">
   <property name="text">
    <string>信息修改</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>printbtn</tabstop>
  <tabstop>findbtn</tabstop>
  <tabstop>information</tabstop>
  <tabstop>list_2</tabstop>
  <tabstop>printbtn_2</tabstop>
  <tabstop>totalbtn</tabstop>
  <tabstop>totaltext</tabstop>
  <tabstop>name_3</tabstop>
  <tabstop>incar</tabstop>
  <tabstop>number</tabstop>
  <tabstop>cargoods</tabstop>
  <tabstop>totalprice</tabstop>
  <tabstop>paytotal</tabstop>
  <tabstop>printbtn_3</tabstop>
  <tabstop>tableView_2</tabstop>
  <tabstop>findbtn_2</tabstop>
  <tabstop>changebtn</tabstop>
  <tabstop>deletebtn</tabstop>
  <tabstop>goodname_2</tabstop>
  <tabstop>barcode_2</tabstop>
  <tabstop>amount_2</tabstop>
  <tabstop>inprice</tabstop>
  <tabstop>outprice</tabstop>
  <tabstop>sup_2</tabstop>
  <tabstop>goodname</tabstop>
  <tabstop>barcode</tabstop>
  <tabstop>amount</tabstop>
  <tabstop>inPrice</tabstop>
  <tabstop>outPrice</tabstop>
  <tabstop>sup</tabstop>
  <tabstop>input</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
